CovidTracker/Pages/Index.cshtml.cs

```diff
@@ -22,9 +22,20 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
-                string jsonString = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+                string covid_data = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+                string result = "";
+
+                try
+                {
+                    result = webClient.DownloadString(covid_data);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling API", e);
+
+                }
```

Comment:
Logging is good, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute as if nothing went wrong.  That might not be the desired behavior.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().